persistence/sql/persister_session.go

```diff
@@ -47,7 +47,7 @@ func (p *Persister) ListSessionsByIdentity(ctx context.Context, iID uuid.UUID, a
 	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.ListSessionsByIdentity")
 	defer span.End()
 
-	s := make([]*session.Session, 0)
+	var s []*session.Session
```

Comment:
This should not change the behavior, the function should previously already return an empty slice.
https://go.dev/play/p/KbJYKpXGsLo